source/js/components/join/join.jsx

```diff
@@ -53,7 +54,9 @@ export default class JoinUs extends React.Component {
       navWrapper.querySelector(".form-dismiss") &&
       this.state.apiSuccess
     ) {
-      navWrapper.querySelector(".form-dismiss").textContent = "Back to menu";
+      navWrapper.querySelector(".form-dismiss").textContent = (
+        <Localized id="back-to-menu">{`Back to menu`}</Localized>
```

Comment:
I'll take a closer look tomorrow; for now I wanted to quickly ask about this: wouldn't this assign `[object Object]` to `textContent`? AFAIK, the return value of `React.createElement`, even for function components`, is a plain object representing the component instance?

Reply:
I thought I tested that, but you’re absolutely right

Reply:
what is this `id` for?